overlord/snapstate/snapstate_test.go

```diff
@@ -231,7 +235,10 @@ func verifyUpdateTasks(c *C, opts, discards int, ts *state.TaskSet, st *state.St
 	expected = append(expected,
 		"set-auto-aliases",
 		"setup-aliases",
+		"run-hook",
```

Comment:
That's a very neat idea, I've done that, readability of these tests is much improved now (plus it gives us better confidence about all the hooks). Thanks!